client/asset/btc/btc.go

```diff
@@ -1853,40 +1852,43 @@ func (btc *baseWallet) splitOption(req *asset.PreSwapForm, utxos []*compositeUTX
 		btc.log.Errorf("estimateSwap (with split) error: %v", err)
 		return nil
 	}
-	if !splitUsed {
-		// unable to do the split. no option.
-		btc.log.Debugf("split option unavailable")
-		return nil
-	}
 	symbol := strings.ToUpper(btc.symbol)
 
-	xtraFees := splitEst.RealisticWorstCase - noSplitEst.RealisticWorstCase
-	pctChange := (float64(splitEst.RealisticWorstCase)/float64(noSplitEst.RealisticWorstCase) - 1) * 100
-	overlock := noSplitLocked - splitLocked
-
-	var reason string
-	if pctChange > 1 {
-		reason = fmt.Sprintf("+%d%% fees, -%s %s overlock", int(math.Round(pctChange)), prettyBTC(overlock), symbol)
-	} else {
-		reason = fmt.Sprintf("+%.1f%% fees, -%s %s overlock", pctChange, prettyBTC(overlock), symbol)
-	}
-
-	desc := fmt.Sprintf("Using a split transaction to prevent temporary overlock of %s %s, but for additional fees of %s %s",
-		prettyBTC(overlock), symbol, prettyBTC(xtraFees), symbol)
-
-	return &asset.OrderOption{
+	opt := &asset.OrderOption{
 		ConfigOption: asset.ConfigOption{
 			Key:           splitKey,
 			DisplayName:   "Pre-size Funds",
-			Description:   desc,
-			DefaultValue:  btc.useSplitTx(),
 			IsBoolean:     true,
+			DefaultValue:  false, // not nil interface
 			ShowByDefault: true,
 		},
-		Boolean: &asset.BooleanConfig{
-			Reason: reason,
-		},
+		Boolean: &asset.BooleanConfig{},
 	}
+
+	if !splitUsed || splitLocked >= noSplitLocked { // locked check should be redundant
+		opt.Boolean.Reason = fmt.Sprintf("avoids no %s overlock for this order (ignored)", symbol)
+		opt.Description = fmt.Sprintf("A split transaction for this order avoids no %s overlock, "+
+			"but adds additional fees.", symbol)
+		return opt // do not show by default, but explain why
```

Comment:
Rather than having a missing option, I believe it is a better UX to show it, unchecked, and with an appropriate reason and description.  Otherwise, the user is wondering where the option went that they normally use.